packages/mobile/src/screens/profile-screen/ArtistRecommendations/ArtistRecommendations.tsx

```diff
@@ -102,7 +102,7 @@ export const ArtistRecommendations = (props: ArtistRecommendationsProps) => {
   })
 
   const suggestedArtists = useProxySelector(
```

Comment:
need proxy selector? just cause it's expensive and really needed when rerenders happen all the time